class PreparerException(Exception):
  pass

class Preparer(object):
  def prepare(self):
    raise Exception()
